keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -267,26 +271,33 @@ How will UX be reviewed, and by whom?
-    - If containerd changes are not supported, host network mode will not be enabled.This would restrict the scenarios that privileged containers would enable, as CNI plugins, network policy, etc. rely on host network mode to enable access to host network resources.
-    - If CRI changes to enable a privileged flag are not supported, there would be a less-ideal workaround via annotations in the pod container spec. 
-    - The CRI changes may make an annotation in the OCI spec until the OCI updates are included.
+      - Adding `WindowsPodSandboxConfig` and `WindowsSandboxSecurityContext` message
+      - Adding `host_process` flag to `WindowsContainerSecurityContext`
+      - Pass security context and flag of runtime spec to podsandbox spec (not currently supported, open issue: https://github.com/kubernetes/kubernetes/issues/92963)
+  - Kubelet: Pass host_process flag and windows security context options to runtime spec.
+- There are risks that changes at each of these levels may not be supported.
+  - If containerd changes are not supported, host network mode will not be enabled.This would restrict the scenarios that privileged containers would enable, as CNI plugins, network policy, etc. rely on host network mode to enable access to host network resources.
+  - If CRI changes to enable a privileged flag are not supported, there would be a less-ideal workaround via annotations in the pod container spec.
+  - The CRI changes may make an annotation in the OCI spec until the OCI updates are included.
+
+### PSP support
+
+For alpha we will update [Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/) with information on the new `hostProcess` flag.
 
-### PSP support 
-Additionally, privileged containers may impact other pod security policies (PSPs) outside of allowPrivilegedEscalation. We will provide guidance similar to [Pod Security Standards](https://v1-18.docs.kubernetes.io/docs/concepts/security/pod-security-standards/) for Windows privileged containers.  There is an [analysis for non-privileged containers](https://github.com/kubernetes/kubernetes/issues/64801#issuecomment-668897952) which can be augmented with the details below.  There is an open question if PSP should be updated or the recommendation is to use an out of tree enforcement tool such as GateKeeper.  The anticipated impacted PSPs include:
+Additionally, privileged containers may impact other pod security policies (PSPs) outside of allowPrivilegedEscalation. We will provide guidance similar to [Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/) for Windows privileged containers when graduating this feature out of alpha. There is an [analysis for non-privileged containers](https://github.com/kubernetes/kubernetes/issues/64801#issuecomment-668897952) which can be augmented with the details below.The anticipated impacted PSPs include:
```

Comment:
heard PSP will be deprecated soon. Will this design be affected?

Reply:
see https://github.com/kubernetes/enhancements/pull/2288#discussion_r572591452 :)